src/components/Hero.jsx

```diff
@@ -1,17 +1,126 @@
 import React from 'react'
+import styled from 'styled-components'
 
-const Hero = ({ eyebrowText, mainHeading, content, img, imgDescription }) => {
+const StyledHeroSection = styled.section`
+  background-color: #b3ddf2;
```

Comment:
Let's use a CSS variable for this. If it doesn't exist let's add it to the global styles.